MAINTAINING.md

```diff
@@ -0,0 +1,59 @@
+# Maintaining React-Bootstrap
+
+This document is primarily for members of the BlessCSS organization, though
+publicly available for your viewing pleasure. It describes how we work as a
+team to provide our downstream users with a great product. You will find
+descriptions for common tasks such as triaging, merging pull requests, and release
+procedures.
+
+If you are interested in contributing to Bless, you should check out the
+[Contributing Guide](./CONTRIBUTING.md).
+
+## Triaging Issues
+
+New issues pop up, We need to identify urgent issues (such as nobody can use the
```

Comment:
@paulyoung The sentence was newlined to force text wrapping. When looking at the file raw the content in this section might not get wrapped correctly, which is the case in some editors.